sql/core/src/test/scala/org/apache/spark/sql/sources/InsertSuite.scala

```diff
@@ -1255,6 +1255,173 @@ class InsertSuite extends DataSourceTest with SharedSparkSession {
     }
   }
 
+  test("SPARK-38811 INSERT INTO on columns added with ALTER TABLE ADD COLUMNS: Positive tests") {
+    // There is a complex expression in the default value.
+    withTable("t") {
+      sql("create table t(i boolean) using parquet")
```

Comment:
Let's assign "create table t(i boolean) using parquet" to a `val`

Reply:
Done, and same for `create table t(i int) using parquet` as well.